### PR TITLE
removed how to contribute section from readme

### PR DESCRIPTION
**Pull request by AnsurM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,44 +10,6 @@ Our main motivation for creating this repository links back to a [LinkedIn group
 4. Inside the folder with their name, participant has to make a `new file` for each `challenge` that they want to solve and solve the challenge in it.
 5. Once they are done solving the challenges, they will make a `pull request` in to the relevant `branch` where they had solved the challenges.
 6. After some safety and consistency checks, one of the `contributors` of this repository will accept the  `pull request`, hence, the participant will also have their code included in the repository.
-
```
Comment (asadkhalid305) [inline README.md:13]: Why did you remove this code?

Comment (AnsurM) [reply README.md:13]: this section has been added to the contributors.md
